distributions/pom.xml

```diff
@@ -16,7 +16,6 @@
   <packaging>pom</packaging>
 
   <modules>
-    <module>openhab-addons</module>
```

Comment:
This should not be removed here - "addons", "addons-legacy" and "openhab" are our distributables, so they should be within this folder.

Reply:
See below